app/marketing/tests/management/commands/test_assemble_leaderboards.py

```diff
@@ -0,0 +1,131 @@
+            network='net',
+            expires_date=date.today() + timedelta(days=1),
+            tokenAddress='0x0000000000000000000000000000000000000000'
+        )
+        fulfiller_profile = Profile.objects.create(
+            data={},
+            handle='fred',
+            email='fred@bar.com'
+        )
+        bounty = Bounty.objects.create(
+            title='foo',
+            value_in_token=3,
+            token_name='USDT',
+            web3_created=datetime(2008, 10, 31),
+            github_url='https://github.com/gitcoinco/web',
+            token_address='0x0',
+            issue_description='hello world',
+            bounty_owner_github_username='flintstone',
+            is_open=False,
+            accepted=True,
+            expires_date=datetime(2008, 11, 30),
+            idx_project_length=5,
+            project_length='Months',
+            bounty_type='Feature',
+            experience_level='Intermediate',
+            raw_data={},
+            idx_status='submitted',
+            current_bounty=True
+        )
+        bounty_fulfillment = BountyFulfillment.objects.create(
+            fulfiller_address='0x0000000000000000000000000000000000000000',
+            fulfiller_email='',
+            fulfiller_github_username='fred',
+            fulfiller_name='Fred',
+            bounty=bounty,
+            profile=fulfiller_profile,
+        )
+
+        self.bounty = bounty
+        self.tip = tip
+
+    def test_sum_tips(self):
+        """Test sum tips of assemble leaderboards."""
+
+        sum_tips(self.tip, ['john'])
+        print(assemble_leaderboards.ranks)
+        assert assemble_leaderboards.ranks['all_fulfilled']['john'] == 7
+        assert assemble_leaderboards.ranks['all_earners']['john'] == 7
+
+        assert assemble_leaderboards.ranks['weekly_fulfilled']['john'] == 7
+        assert assemble_leaderboards.ranks['weekly_earners']['john'] == 7
+        assert assemble_leaderboards.ranks['weekly_all']['john'] == 7
+
+        assert assemble_leaderboards.ranks['monthly_fulfilled']['john'] == 7
+        assert assemble_leaderboards.ranks['monthly_earners']['john'] == 7
+        assert assemble_leaderboards.ranks['monthly_all']['john'] == 7
+
+        assert assemble_leaderboards.ranks['yearly_fulfilled']['john'] == 7
+        assert assemble_leaderboards.ranks['yearly_earners']['john'] == 7
+        assert assemble_leaderboards.ranks['yearly_all']['john'] == 7
+
+    def test_sum_bounties(self):
+        """Test sum tips of bounties leaderboards."""
+
+        sum_bounties(self.bounty, ['fred'])
+
+        print(assemble_leaderboards.ranks)
+
+        assert assemble_leaderboards.ranks['all_all']['fred'] == 3
+        assert assemble_leaderboards.ranks['weekly_all']['fred'] == 3
+        assert assemble_leaderboards.ranks['monthly_all']['fred'] == 3
+        assert assemble_leaderboards.ranks['yearly_all']['fred'] == 3
+
+        assert len(assemble_leaderboards.ranks['weekly_fulfilled']) == 0
```

Comment:
Opposed to `len()`, can you use either `assert any(dict) is False` or `assert not dict`